src/components/drawer/product-filter-drawer/ProductFilterDrawer.jsx

```diff
@@ -6,15 +6,21 @@ import {
   PriceSortFilter,
   PriceRangeFilter,
 } from '../../product-filter';
+import * as ACTIONS from '../../../shared/actions/types';
+import { useProductFilter } from '../../../shared/context/product-filter-context';
 
 export const ProductFilterDrawer = () => {
+  const { dispatch } = useProductFilter();
```

Comment:
The dispatch could be named a bit more precisely. Because if you have more than one context then it could create confusion in the naming of dispatches. It could be named something as `productFilterDispatch` or `filterDispatch`